### ISSUE TITLE
[INFO]

### ISSUE THREAD

**Issue by sh1man999** [emgucv/emgucv#836]
When will this work for .Net 7.0 ?
I can't build the build on debian, although it works on windows for .Net core

**Comment by emgucv** [comment]
What version of Emgu CV debian 64 runtime are you using?

Please use this instruction to check if you are missing any dependencies: 
https://www.emgu.com/wiki/index.php/Download_And_Installation#System.DllNotFoundException

The debian package is build using debian bullseye (11). 

If you are using the 4.7 release, you may need to run this script in your docker image to install all the dependencies:
https://github.com/emgucv/emgucv/blob/4.7.0/platforms/debian/bullseye/apt_install_dependency

**Comment by sh1man999** [comment]
yes, I'm using version 4.7 and your answer of installing dependencies helped me

**Comment by emgucv** [comment]
Closing as resolved — installing the native dependencies via the `apt_install_dependency` script fixed it. For anyone hitting this: the `Emgu.CV.runtime.debian-*` NuGet packages ship `libcvextern.so` but its native dependencies (OpenCV, FFmpeg, etc.) must be present on the system. Run the appropriate script for your Debian version, e.g. `platforms/debian/bullseye/apt_install_dependency` for Debian 11, or `platforms/debian/bookworm/apt_install_dependency` for Debian 12.
